stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        owl21_reduced[kept_columns].drop_duplicates().reset_index().drop(columns="index")
</pre><p> </p><p><code>.reset_index(drop=True)</code> would do the same as <code>.reset_index().drop(columns="index")</code> I believe , if you'd like to make the line a bit shorter</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done</p>